test/writer-tests.js

```diff
@@ -3,28 +3,28 @@ var Writer = require(__dirname + "/../");
 var assert = require('assert');
 var util = require('util');
 
-assert.equalBuffers = function(actual, expected) {
-  var spit = function(actual, expected) {
+assert.equalBuffers = function (actual, expected) {
+  var spit = function (actual, expected) {
     console.log("");
     console.log("actual " + util.inspect(actual));
     console.log("expect " + util.inspect(expected));
     console.log("");
   };
-  if(actual.length != expected.length) {
+  if (actual.length != expected.length) {
     spit(actual, expected);
     assert.equal(actual.length, expected.length);
   }
-  for(var i = 0; i < actual.length; i++) {
-    if(actual[i] != expected[i]) {
+  for (var i = 0; i < actual.length; i++) {
+    if (actual[i] != expected[i]) {
       spit(actual, expected);
     }
-    assert.equal(actual[i],expected[i]);
+    assert.equal(actual[i], expected[i]);
```

Comment:
`assert.equal` is deprecated (in favor of `assert.strictEqual`), I can fix this in another PR if you want though

Reply:
ah no worries, just changed in this PR just now